docs/source/getting-started/installation.rst

```diff
@@ -0,0 +1,132 @@
+managed on-prem solution. Check out 
+`the different tiers <https://deepchecks.com/pricing>`__ if you want 
+more info about the licensed versions, or contact us at info@deepchecks.com.
+
+There are two parts to the system:
+
+- The client side (``deepchecks-client`` Python package), for interacting with the 
+  monitoring app
+- The monitoring service itself (UI, backend, etc.), which is built of several 
+  docker containers and deployed with docker-compose.
+  You need to install it only if you're using the open-source self-hosted version.
+
+The installation for each of the two parts is independent, and this guide includes a step-by-step tutorial for both options.
+Step #2 is the local installation of the monitoring service, which you'll skip if you're not working with the open-source deployment.
+
+
+Installation Steps
+=====================
+
+In order to work with Deepchecks Monitoring, you need to:
+
+1. :ref:`Install the deepchecks-client with pip <installation__deepchecks_client>`, which installs the SDK for interacting
+   with the app, along with the :doc:`Deepchecks Testing <deepchecks:getting-started/welcome>` package.
+2. (Only for open source deployments) :ref:`Deploy the self-hosted open source <installation__deepchecks_self_hosted>` 
+   instance of Deepchecks Monitoring app 
+3. Log in to the Deepchecks Monitoring app and 
+   :ref:`Create an organization or join an existing organization <installation__create_or_join_organization>`
+4. :ref:`Obtain your API key <installation__obtain_api_key>` from the app
+
+
+
+.. _installation__deepchecks_client:
+
+Step #1: Install ``deepchecks-client`` with pip
+====================================================
+
+deepchecks-client is a Python package that installs the SDK for interacting with the monitoring app and
+the :doc:`Deepchecks Testing <deepchecks:getting-started/welcome>` package, which can be useful when exploring existing issues
+(e.g. when using the "download" feature to further research the issue in your local environment)
+
+It can be installed using pip:
+
+.. code-block:: bash
+
+    pip install deepchecks-client --upgrade
+
+
+.. _installation__deepchecks_self_hosted:
+
+Step #2: Deploy the self-hosted open source
+==============================================
+
+If you're using the SaaS version, you can skip this step.
+If you wish to self-host the deepchecks monitoring app, jump over to the :ref:`self-host deepchecks monitoring <installation__self_host_deepchecks>` installation instructions,
+and continue to the next step once you've completed the local installation and can open your local deepchecks monitoring instance.
+
+
+.. _installation__create_or_join_organization:
+
+Step #3: Create or join an organization
+=============================================
+
+For SaaS usage: following an invite mail (sign up to our invite system at the `Deepchecks Website`_, or contact us at info@deepchecks.com),
```

Comment:
In the whole doc the split is open-source / Saas . Would keep it here as well for consistency